recipe/meta.yaml

```
@@ -33,7 +33,7 @@ requirements:
 
 test:
   imports:
-    - dbt
+    - dbt.adapters.spark
```

Let's make that test additive
```suggestion
    - dbt
    - dbt.adapters.spark
```

---

OK, I see now that the `dbt` import wasn't really testing anything, so this should be fine as-is